### PR TITLE
Add improvements from review suggestions

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: checkout/checkout_service.bal [hunk 1]
```diff
@@ -15,10 +15,11 @@
 // under the License.
 
 import ballerina/grpc;
-import ballerina/uuid;
 import ballerina/log;
+import ballerina/uuid;
 import ballerinax/jaeger as _;
 import wso2/client_stubs as stub;
+import wso2/money_utils as utils;
```
Comment (MaryamZi) [inline checkout/checkout_service.bal:22]: How about using `money` as the prefix instead?

File: email/email_service.bal [hunk 2]
```diff
@@ -60,43 +60,38 @@ service "EmailService" on new grpc:Listener(9097) {
...
+        log:printError("Error sending confirmation mail ", sendMessageResponse);
         return sendMessageResponse;
     }
+}
 
-    function getConfirmationHtml(stub:OrderResult result) returns xml|error {
-        string fontUrl =
+function getConfirmationHtml(stub:OrderResult result) returns xml|error {
+    string fontUrl =
                     "https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap";
```
Comment (MaryamZi) [inline email/email_service.bal:80]: Should this be a constant?

File: frontend/rpc.bal [hunk 3]
```diff
@@ -86,106 +86,92 @@ final stub:CheckoutServiceClient checkoutClient = check new (string `http://${ch
 isolated function getSupportedCurrencies() returns string[]|grpc:Error {
     stub:GetSupportedCurrenciesResponse|grpc:Error supportedCurrencies = currencyClient->GetSupportedCurrencies({});
     if supportedCurrencies is grpc:Error {
-        log:printError("failed to call getSupportedCurrencies from currency service", 'error = supportedCurrencies);
+        log:printError("failed to get supported currencies from currency service", supportedCurrencies);
```
Comment (MaryamZi) [inline frontend/rpc.bal:89]:
```suggestion
        log:printError("failed to get supported currencies from the currency service", supportedCurrencies);
```

May be applicable in other places too.

File: money_utils/money_utils.bal [hunk 4]
```diff
@@ -13,9 +13,10 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
 import wso2/client_stubs as stub;
 
-const map<string> logos = {
+public const map<string> LOGOS = {
```
Comment (MaryamZi) [inline money_utils/money_utils.bal:19]: Can/should this be `CURRENCY_SYMBOLS`?

https://en.wikipedia.org/wiki/Currency_symbol

Comment (dilanSachi) [reply money_utils/money_utils.bal]: Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b

File: frontend/rpc.bal (hunk 3 above)
Comment (dilanSachi) [reply frontend/rpc.bal]: Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b

File: email/email_service.bal (hunk 2 above)
Comment (dilanSachi) [reply email/email_service.bal]: Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b

File: checkout/checkout_service.bal (hunk 1 above)
Comment (dilanSachi) [reply checkout/checkout_service.bal]: Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b

**Comment by dilanSachi** [comment]
Ill update the logs and return error types in a separate pr

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**